ogcapi-features/README.md

```diff
@@ -62,7 +66,7 @@ The core OGC API - Features endpoints are shown below, with details provided in
 | Endpoint                                        | Returns                                                 | Description                                                                                                                                                             |
 | ----------------------------------------------- | ------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `/`                                             | [Catalog](../stac-spec/catalog-spec/README.md)          | Landing page, links to API capabilities                                                                                                                                 |
-| `/api`                                          | OpenAPI 3.0 JSON                                        | Returns an OpenAPI description of the service from the `service-desc` link `rel`. The path for this endpoint is only recommended to be `/api`, but may be another path. |
+| `/api`                                          | any                                                     | Returns an OpenAPI description of the service from the `service-desc` link `rel`. The path for this endpoint is only recommended to be `/api`, but may be another path. |
```

Comment:
OpenAPI is wrong, it could be anything. Probably also an issue in several other places, did only realize it now...

```suggestion
| `/api`                                          | any                                                     | Returns a machine-readable API description of the service from the `service-desc` link `rel`. The path for this endpoint is only recommended to be `/api`, but may be another path. |
```